### ISSUE TITLE
Copy button on doc examples copies line numbers as well as code

### ISSUE THREAD

**Issue by ramalho** [pyscript/pyscript#565]
### Checklist

- [X] I added a descriptive title
- [X] I searched for other issues and couldn't find a solution or duplication
- [X] I already searched in Google and didn't find any good information or help

### What happened?

The code box used in [this example](https://docs.pyscript.net/latest/concepts/what-is-pyscript.html#example) includes a button on the top right to copy the source code. However, when you paste the code anywhere, it comes with the line numbers, making it unusable.

Example pasted below:

```html
 1<html>
 2    <head>
 3      <link rel="stylesheet" href="https://pyscript.net/alpha/pyscript.css" />
 4      <script defer src="https://pyscript.net/alpha/pyscript.js"></script>
 5      <style>
 6        .pulse {
 7            animation: pulse 2s cubic-bezier(0.4, 0, 0.6, 1) infinite;
 8        }
 9
10        @keyframes pulse {
11          0%, 100% {
12            opacity: 1;
13          }
14          50% {
15            opacity: .2;
16          }
17        }
18      </style>
19      <py-env>
20        - numpy
21        - matplotlib
22      </py-env>
23    </head>
24
25  <body>
26    <h1>Let's plot random numbers</h1>
27    <div id="plot">
28      <div class="pulse" >
29        <p style='font-family: monospace sans-serif;'><big><big><big><big>&#10096;py&#10097;</big></big></big></big></p>
30      </div>
31    </div>
32    <py-script>
33import matplotlib.pyplot as plt
34import numpy as np
35
36x = np.random.randn(1000)
37y = np.random.randn(1000)
38
39fig, ax = plt.subplots()
40ax.scatter(x, y)
41pyscript.write('plot', fig)
42    </py-script>
43  </body>
44</html>
```

### What browsers are you seeing the problem on? (if applicable)

Chrome

### Additional Context

As a work-around the user can manually select the source code and copy.

**Comment by ramalho** [comment]
I just noticed that the copy button appears only when the cursor is hovering the code box. It would be better if the copy button was always visible when this issue is fixed.

**Comment by nmstoker** [comment]
With the current way the copy button is implemented, I believe a solution here will depend on either:

1. this open issue: https://github.com/executablebooks/sphinx-copybutton/pull/138
2. Removing the line numbers, here: https://github.com/pyscript/pyscript/blob/ee9b0960f75dfc8ac4cf538c79813787bdf2955d/docs/concepts/what-is-pyscript.md?plain=1#L22

**Comment by jmaurodev** [comment]
If you read the open issue [Exclude unselectable text from being copied](https://github.com/executablebooks/sphinx-copybutton/pull/138) @nmstoker just brought, you may notice:

1. It was started on 20 Sep 2021 and today, 4 July 2022, it have already completed 9 months.
2. Judgind by issue's actual state, it's wise to admit that it may not be ready soon.
![Captura de tela de 2022-07-04 12-37-16](https://user-images.githubusercontent.com/13485954/177185630-8a0d2ec9-037b-4dea-a327-e36c434ebb1c.png)

Looks like the guaranteed solution is the deletion of 
https://github.com/pyscript/pyscript/blob/ee9b0960f75dfc8ac4cf538c79813787bdf2955d/docs/concepts/what-is-pyscript.md?plain=1#L22
as @nmstoker suggested.

**Comment by marimeireles** [comment]
Closing as fixed by #745
If you think it's not a good solution, please feel free to reopen.
